rllib/agents/ppo/tests/test_ppo.py

```diff
@@ -82,6 +82,9 @@ def test_ppo_compilation_and_lr_schedule(self):
         # Settings in case we use an LSTM.
         config["model"]["lstm_cell_size"] = 10
         config["model"]["max_seq_len"] = 20
+        # Use default-native keras model whenever possible.
```

Comment:
Testing, whether this flag works (try using native keras default model).
Currently only works for FCNets.